extensions/extensions-helm/pom.xml

```diff
@@ -105,18 +110,39 @@
                                 <goal>dependency-update</goal>
                             </goals>
                         </execution>
+                        <!--
+                            dry run doesn't seem necessary for normal ops, but you can add it
+                            by uncommenting this out, if needed:
+                        <execution>
+                            <id>simulates-an-install</id>
+                            <phase>integration-test</phase>
+                            <goals>
+                                <goal>dry-run</goal>
+                            </goals>
+                        </execution>
+                        -->
+                        <execution>
+                            <id>deploy</id>
+                            <phase>deploy</phase>
+                            <goals>
+                                <goal>push</goal>
+                            </goals>
+                        </execution>
                     </executions>
                     <configuration>
-                        <skipInit>true</skipInit>
                         <appVersion>${project.version}</appVersion>
                         <chartDirectory>${project.basedir}</chartDirectory>
                         <namespace>default</namespace>
                         <templateGenerateName>true</templateGenerateName>
                         <skip>${skip.helm}</skip>
-                        <skipTemplate>false</skipTemplate>
                         <skipDependencyBuild>true</skipDependencyBuild>
+                        <!-- For OCI, do not init: -->
+                        <skipInit>false</skipInit>
```

Comment:
Q: There seems to be a mismatch between the comment and the code here. Which do we mean?

Reply:
I think this should be set to true right?

Reply:
Yep - my bad.  We DO need the init, but it needs to NOT have ghcri.io.  Will remove the comment.